nasa/earthdata-search
# EDSC-4020: Removed error banner for missing colormaps.

# Overview

### What is the feature?

Removed the banner for missing colormaps.

### What is the Solution?

Removed the dispatching of an error when a colormap is fails to download.

### What areas of the application does this impact?

List impacted areas.

# Testing

### Reproduction steps

- **Environment for testing:**prod
- **Collection to test with:**C2021957657-LPCLOUD

1. go to http://localhost:8080/search/granules?p=C2021957657-LPCLOUD
2. observe no banner showing up

### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist
